Makefile

```diff
@@ -40,7 +40,17 @@ install: build
 	@echo "    }"
 	@echo "  }"
 	@echo "}"
-	
+
+install-tf-0.12: build
+	@echo "✓ Installing provider for Terraform 0.12 ..."
+	@mkdir -p '$(HOME)/.terraform.d/plugins/$(shell go version | awk '{print $$4}' | sed 's#/#_#')/'
```

Comment:
tf12 doesn't actually need any dir version prefix. look at installation instructions:

```
curl https://raw.githubusercontent.com/databrickslabs/databricks-terraform/master/godownloader-databricks-provider.sh | bash -s -- -b $HOME/.terraform.d/plugins
```

my testing setup is: `$PATH` has 0.12 binary, terraform13 and terraform14 for other version as well.